src/tools/kata-ctl/src/utils.rs

```diff
@@ -30,4 +49,48 @@ mod tests {
         let res = drop_privs();
         assert!(res.is_ok());
     }
+
+    #[test]
+    fn test_kernel_version_empty_input() {
```

Comment:
@amshinde I would approve as-is but these tests are suitable for a [table-driven test](https://github.com/kata-containers/kata-containers/blob/main/docs/Unit-Test-Advice.md#rust-table-driven-tests)

Reply:
@cmaf some of these tests require a different setup such as creating a temp directory while others dont. This makes table driven test quite awkward. I could do it for 2 of the tests, but saw it as a overhead for just 2 tests.